tests/tensorflow/test_keras_model_export.py

```diff
@@ -309,7 +309,7 @@ def __call__(self):
         model_path, keras_model_kwargs={"custom_objects": correct_custom_objects}
     )
     assert model_loaded is not None
-    if Version(tf.__version__) <= Version("2.11.0"):
+    if Version(tf.__version__) <= Version("2.11.0") or Version(tf.__version__) > Version("2.15.0"):
```

Comment:
Can we change the comment in `else` branch to reflect the version check > 2.15.0?